website/client/components/inventory/stable/mountItem.vue

```diff
@@ -15,6 +15,11 @@ div
     slot(name="popoverContent", :item="item")
 </template>
 
+<style lang="scss">
```

Comment:
since this style block is not scoped can you move it to the assets/scss folder?

Reply:
It's also a duplicate of the other change you made so you should just keep one version and move it to assets/scss

Reply:
when I move to assets/scss is that going to be a new file and if so how do I call it? Is it called automatically from some build task?

Reply:
I would put it into the existing item.scss file